app/models/setting/general.rb

```diff
@@ -24,7 +25,7 @@ def self.default_settings
       set('fix_db_cache', N_('Fix DB cache on next Foreman restart'), false, N_('Fix DB cache')),
       set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
       set('proxy_request_timeout', N_("Open and read timeout for HTTP requests from Foreman to Smart Proxy (in seconds)"), 60, N_('Smart Proxy request timeout')),
-      set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text')),
+      set('login_text', N_("Text to be shown in the login-page footer"), "Version #{version}", N_('Login page footer text')),
```

Comment:
It would be better to have a constant
```suggestion
      set('login_text', N_("Text to be shown in the login-page footer"), "Version $VERSION", N_('Login page footer text. Use $VERSION to be replace by current version.')),
```